Linking in prerequisite sections

rudri182 opened the pull request:
I know there are so many commits and there can be some duplication in commits. But I was very confused doing cherry picking. So let me know whether I have done correctly or not. Cherry-pick from PR #793

rudri182 commented:
If this branch is working properly then may I add the files to this branch?

saberprashant commented:
Hi @rudri182, well, in this PR, you've cherry-picked the changes of only 2 files. But in your PR #793, you changed a total of 15 files. So, my question is, are these your final changes? or you want to do some more changes?

rudri182 commented:
> Hi @rudri182, well, in this PR, you've cherry-picked the changes of only 2 files. But in your PR #793, you changed a total of 15 files. So, my question is, are these your final changes? or you want to do some more changes?

@saberprashant Thanks for noticing it.
Yes, I have more files to add. But I am trying it for the first time. So I was not sure about it. Now I will upload more. 😊

rudri182 commented:
I have tried my best and modified outline as the comments by @praveenscience in the branch **Converting check for understanding** PR #793. And added the questions in the Stack.md which is the main task for issue #700. Please review it.

rudri182 commented:
> Are you changing prerequisites or working on "check for understanding"s? Those are 2 separate issues.

I am working on issue #700 but in PR #793 @praveenscience suggested some changes in prerequisites section that was pushed by mistake. After that, I also tried to remove those files by `git -rm <filename>` and then pushed them but still, they are appearing. I have worked on prerequisites sections previously so I am working on **CFU** section here.

alodahl commented:
Keep trying!  This is a real life problem, and these are great skills to have for an engineering career. Its also a good reminder to check which branch you're on before you commit or push!

rudri182 commented:
I have resolved the conflicts. I will make a new branch for the  **prerequisites** section and upload these there.

rudri182 commented:
May I remove the stack.md file from here and keep other things as it is? So there is no need to create another branch. As I have already created a new branch for **Check For Understanding** section.

alodahl commented on react-js/react.md:
```diff
@@ -4,9 +4,8 @@
-* [Javascript I - VI](https://github.com/Techtonica/curriculum/tree/master/javascript)
-* [jQuery Topic Outline](https://github.com/Techtonica/curriculum/tree/master/jquery)
+* [Javascript I - VI](../javascript)
+* [jQuery Topic Outline](../jquery)
```
Please link to the jquery file

rudri182 replied:
Linking done 👍

Kundan28 commented on deploying/deploying.md:
```diff
@@ -5,20 +5,17 @@
-- [MongoDB](/databases/mongo-db.md)
+Node.js and npm must be installed
+An existing Node.js app
+- [JS I - VI](https://github.com/Techtonica/curriculum/tree/master/javascript)
```
`JS I - VI` link could be made short too...

Kundan28 commented on node-js/node-js.md:
```diff
@@ -5,11 +5,11 @@
+- [How the Internet Works](../requests-and-responses/requests-and-responses.md)
+- [AJAX](../ajax/ajax.md)
+- [Git and Version Control](../git-version-control/git-version-control.md)
+- [jQuery](https://github.com/Techtonica/curriculum/tree/master/jquery)
```
This link couldn't be made short as the `JavaScript I - VI` ?

Kundan28 commented on open-source/open-source.md:
```diff
@@ -4,9 +4,9 @@
 
 ### Prerequisites
+- ["Git and version control" lesson](../git-version-control/git-version-control.md)
 
```
You can remove the blank line number 8.

rudri182 commented:
I have made the changes as suggested by @Kundan28. Please suggest if there are more changes to be made. 😊

rudri182 commented:
👍😊

On Tue, May 7, 2019, 4:21 PM Kundan <notifications@github.com> wrote:

> *@Kundan28* approved this pull request.
>
> Nice work @rudri182 <https://github.com/rudri182> 💯
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Techtonica/curriculum/pull/805#pullrequestreview-234440790>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHRNBVKHDINYCH5UJJHBDDLPUFNJJANCNFSM4HJRVDNA>
> .
>

alodahl commented:
@rudri182 , does this fix all the prereq sections in https://github.com/Techtonica/curriculum/issues/731? if so please rename and describe your issue so we can give you points without confusion.

rudri182 commented:
> @rudri182, does this fix all the prereq sections in #731? if so please rename and describe your
> pr so we can give you points without confusion.

Yes, they are. Still, if there is missing something then I'm happy to solve it. 😊

alodahl commented:
Please fix your title and description.
